conduct/conduct_enforcement.md

```diff
@@ -5,6 +5,13 @@ Committee. It's used when we respond to an issue to make sure we're consistent
 and fair. It should be considered an internal document, but we're publishing it
 publicly in the interests of transparency.
 
+Enforcing the code of conduct with punitive measures or reprimands is an
```

Comment:
@Carreau Suggested edit. Credit to @Matthew-Brett for the bullet points with minor edits by me.

```
Enforcing the Code of Conduct impacts our community today and for the future. It's an
action that we do not take lightly. When reviewing enforcement measures, the Code of
Conduct Committee will keep the following community values in mind:

- act in a personal manner rather than impersonal;
- emphasize empathy for individuals rather than judging behavior;
- increase engagement in good discussion practice;
- better and more explicit support for new members, with the aim of increasing
participation from underrepresented groups in particular.

Our actions will consider that individuals come from different cultural backgrounds and
native languages. These actions will reflect compassion for all individuals. We will seek
to understand, to educate, and, as necessary, take action.
```